lib/sycamore/sycamore/llms/prompts/default_prompts.py

```diff
@@ -119,36 +119,34 @@ class GraphRelationshipExtractorPrompt(SimplePrompt):
 
 class ExtractTablePropertiesPrompt(SimplePrompt):
     user = """
-            You are given a text string where columns are separated by comma representing either a single column, 
+            You are given a text string where columns are separated by comma representing either a single column,
             or a multi-column table each new line is a new row.
             Instructions:
-            1. Parse the table and return a flattened JSON object representing the key-value pairs of properties 
+            1. Parse the table and return a flattened JSON object representing the key-value pairs of properties
             defined in the table.
-            2. Do not return nested objects, keep the dictionary only 1 level deep. The only valid value types 
+            2. Do not return nested objects, keep the dictionary only 1 level deep. The only valid value types
```

Comment:
Grammar - comma splice (Do not return nested objects, keep the dictionary only 1 level deep -> Do not return nested objects; keep the dictionary only 1 level deep)